src/ui/save-html.js

```diff
@@ -1,15 +1,20 @@
 // Module ui/save-html
 // Saves content to HTML when asked to
-import { xmlEscape, removeReSpec } from "core/utils";
-import { pub } from "core/pubsubhub";
 import { ui } from "core/ui";
 import { l10n, lang } from "core/l10n";
+import { pub } from "core/pubsubhub";
+import {
+  toDataURL,
+  makeEPubHref,
+  serialize,
+  exportDocument as newExportDoc,
```

Comment:
We should make `exportDocument()` smarter, we shouldn't need to import `toDataURL`, or `serialize` here. Below, we should just be able to say:

```JS
 get href(){
  return newExportDoc("data-url");
 }
```

And get back the equivalent of `return toDataURL(serialize());`

Reply:
I thought of this:
``` js
const mimeTypes = new Map([
  ["text/html", "html"],
  ["application/xml", "xml"],
]);
export async function exportDocument(mimeType = "text/html", doc = document) {
  const format = mimeTypes.get(mimeType);
  const dataURL = toDataURL(serialize(format, doc), mimeType);
 // ...
}
```

and we get use it as:
``` js
    // newExportDoc === exportDocument
    get href() {
      return newExportDoc(this.type);
    },
```

Reply:
Nice!